composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "shopware/core": "~6.6.0",
-        "swag/migration-assistant": "~14.1"
+        "swag/migration-assistant": "~14.0"
```

Comment:
I think you should also decrease the version of the magento plugin to `9.0.0` again 🙂